### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by SMITH1103**
**Description**:
-- Implemented 400px min width, 950px min height throughout
-- Improvements on selectors to be more compatible with exceptionally small screens
-- Improvements on AssetSelector in dark mode (implemented in Send component)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ specifiers:
   autoprefixer: ^10.3.1
   awilix: ^4.3.4
   bignumber.js: ^9.0.1
+  bip32-path: ^0.4.2
```
Comment (cacampbell) [inline pnpm-lock.yaml:21]: hey... wait a minute. This should be on main and not included here.

File: src/assets/refresh_green.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80.03 77.97"><defs><style>.cls-1{fill:#39ba8b;}</style></defs><path class="cls-1" d="M89.48,48.75l.39-24-7.45,4.63a38.75,38.75,0,0,0-69.32,7,38.2,38.2,0,0,0-1.42,4.76L19.79,43a31,31,0,0,1,1.12-3.75,30.38,30.38,0,0,1,54.42-5.42l-7.52,4.68Z" transform="translate(-9.84 -11.02)"/><path class="cls-1" d="M10.22,51.27l-.38,24,7.44-4.63a38.75,38.75,0,0,0,69.33-7A40.54,40.54,0,0,0,88,58.89L79.91,57a29.81,29.81,0,0,1-1.12,3.75,30.4,30.4,0,0,1-54.42,5.42l7.53-4.68Z" transform="translate(-9.84 -11.02)"/></svg>
```
Comment (cacampbell) [inline src/assets/refresh_green.svg:1]: already on main

File: src/components/interface/RefreshBalance.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<template>
```
Comment (cacampbell) [inline src/components/interface/RefreshBalance.vue:1]: already on main

File: src/components/interface/TotalBalance.vue [hunk 4]
```diff
@@ -2,7 +2,8 @@
   <div class="mt-5 mb-5 text-base text-black-out dark:text-white">
     {{ $t("InterfaceTotalBalance.totalvalue") }}
   </div>
-  <div class="flex items-center">
+
```
Comment (cacampbell) [inline src/components/interface/TotalBalance.vue:5]: already on main

File: src/domain/wallet/hardware-ledger.ts [hunk 5]
```diff
@@ -2,13 +2,17 @@ import { Buffer } from "buffer";
 
 import type { PublicKey } from "@hashgraph/sdk";
```
Comment (cacampbell) [inline src/domain/wallet/hardware-ledger.ts:3]: already on main

File: src/pages/404.vue [hunk 6]
```diff
@@ -14,7 +14,7 @@
       </div>
 
       <div class="mt-10 text-3xl">
-        {{ $("Pages.404.Sorry") }}
+        {{ $t("Pages.404.Sorry") }}
```
Comment (cacampbell) [inline src/pages/404.vue:17]: already on main

File: src/pages/interface/ConvertUnits.vue [hunk 7]
```diff
@@ -28,7 +33,10 @@
       </div>
 
       <div class="self-center block-center">
-        <img src="../../assets/swap.svg" alt="Swap" />
+        <img
+          src="../../assets/swap.svg"
+          alt="Swap"
```
Comment (cacampbell) [inline src/pages/interface/ConvertUnits.vue:38]: translate alt text

File: src/pages/interface/TransactionDetails.vue [hunk 8]
```diff
@@ -43,7 +43,7 @@
       :src="checkmark"
       alt="green checkmark"
     >
-    {{ $("InterfaceTransactionDetails.status.value.success") }}
+    {{ $t("InterfaceTransactionDetails.status.value.success") }}
```
Comment (cacampbell) [inline src/pages/interface/TransactionDetails.vue:46]: already on main

Comment (SMITH1103) [reply src/pages/interface/TransactionDetails.vue]: @cacampbell That probably came from main, I merged main into this branch recently.

**Pull request merged**